index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">
+    <title>Chocolate Pizza</title>
+  </head>
+  <!-- <div class="header"> -->
+  <section>
+    <img id="logo" src="C:\Users\saulg_000\CodeFellows\201\chocolate-pizza\choco-pizza\chocpizzapics\logo.png" alt="" />
```

Comment:
This filepath only exists on your computer. My computer doesn't have a folder called `saulg_000`, so it doesn't work. This filepath needs to be a relative one from the index.html file to the image, so it should just be 

```html 
<img src="logo.png" alt="logo" id="logo" />
```